kotlin-sqlobject/src/main/kotlin/org/jdbi/v3/sqlobject/kotlin/Extensions.kt

```diff
@@ -1,4 +1,4 @@
-/*
+/**
```

Comment:
Any particular reason to mark these up as javadoc instead of comments?

Reply:
The formatter plugin insists -- if it is not Javadoc style already, it will insert a duplicate license header in Javadoc style.  I'm not sure why this happens for particularly Kotlin source files, maybe we should dig deeper, but this change is necessary for now :/

Reply:
As long as these copyright notices don't show up in the docs, I'm fine with this change.